so101_follower_ros2_bridge:
  ros__parameters:
    port: "/dev/ttyACM1"
    id: "follower"
    calibration_dir: "/home/anton/ros2_ws/src/so101_ros2/so101_ros2_bridge/config/calibration/follower.json"
    use_degrees: true
    max_relative_target: 10 # default is none
    disable_torque_on_disconnect: true
    publish_rate: 50.0

